K2Bridge/Models/Response/IKeyedBucket.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace K2Bridge.Models.Response
+{
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// An interface for a bucket class.
+    /// </summary>
+    public interface IKeyedBucket : IBucket
```

Comment:
Is the plan to use this interface on other buckets as well?

Reply:
Refactored to use a base class.